### PR TITLE
Add a new pie charts counting "in-consensus" nodes.

### PR DESCRIPTION
**Pull request by sickpig**
I.e. BU >= 1.6 and ABC >  0.20.0

### CODE REVIEW

File: app/mixins/nodes-data.js [hunk 1]
```diff
@@ -114,6 +118,25 @@ export default Mixin.create({
...
+  nodesByUserAgentInConsensus: computed('nodes.[]', function() {
+    let byUserAgentInC = {};
+    get(this, 'nodes').forEach((node) => {
+      let userAgent = get(node, 'userAgent') || 'unknown';
+      let userAgentWoEB = userAgent.split('(')[0] + '/';
+      let nodeType = userAgentWoEB.split(':')[0].substr(1);
+      let version  = userAgentWoEB.split(':')[1];
+      version = semver.coerce(version.slice(0, -1));
+      // Assuming all clients but BU and BAC to be in consensus
+      if (((nodeType == 'BUCash') && semver.cmp(version, '>=', CONSTANTS.BUminVer))
```
Comment (bugant) [inline app/mixins/nodes-data.js]: Please, use `===`

Comment (sickpig) [reply app/mixins/nodes-data.js]: sure will do

**Comment by sickpig** [comment]
Thanks for the stamp approval @bugant :smile:

Will close and merge via gitflow

**Comment by sickpig** [comment]
merged to `develop` via gitflow a long time ago. closing now.

**Pull request closed without merging**